src/filter.d.ts

```diff
@@ -0,0 +1,11 @@
+import type { ColorToColor, Mode } from './common';
+
+type Filter = (amt?: number, mode?: Mode) => ColorToColor;
```

Comment:
According to the documentation

> The filters were designed for RGB colors, so the mode parameter is expected to be rgb or lrgb.

Should a type of the `mode` parameter looks like `'rgb' | 'lrgb'` ?

Reply:
I think the question above should be asked to the creator of the library? @danburzo what do you think?